test/libyul/yulOptimizerTests/fullSuite/storage.yul

```diff
@@ -0,0 +1,15 @@
+{
+    sstore(4, 5)
+    sstore(4, 3)
+    sstore(8, sload(4))
+}
+// ====
+// step: fullSuite
+// ----
+// {
+//     {
+//         sstore(4, 5)
+//         sstore(4, 3)
+//         sstore(8, sload(4))
```

Comment:
Couldn't `sload(4)` be replaced by `3`?

Reply:
Oh! I actually did not activate it in the full suite yet (on purpose).